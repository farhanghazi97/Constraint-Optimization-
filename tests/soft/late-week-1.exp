m1:tue 9am
m2:wed 9am
m3:fri 12pm
cost:5
