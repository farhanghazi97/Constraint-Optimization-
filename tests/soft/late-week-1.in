# late week soft constraint coupled with hard constraints
meeting, m1
meeting, m2
meeting, m3
domain, m1, late-week, soft
domain, m1, mon 9am-tue 12pm, hard
domain, m2, late-week, soft
domain, m2, before thu, hard
domain, m3, late-week, soft
domain, m3, afternoon, hard
