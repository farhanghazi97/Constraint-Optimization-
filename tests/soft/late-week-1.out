m1:tue 12pm
m2:wed 11am
m3:fri 4pm
cost:5
